doc/components.md

```diff
@@ -372,12 +379,16 @@ component (`game.add(parallaxComponent`).
 Also, don't forget to add you images to the `pubspec.yaml` file as assets or they wont be found.
 
 The `Parallax` file contains an extension of the game which adds `loadParallax`, `loadParallaxLayer`
-and `loadParallaxImage` so that it automatically uses your game's image cache instead of the global
+, `loadParallaxImage` and `loadParallaxAnimation` so that it automatically uses your game's image cache instead of the global
 one. The same goes for the `ParallaxComponent` file, but that provides `loadParallaxComponent`.
 
 If you want a fullscreen `ParallaxComponent` simply omit the `size` argument and it will take the
 size of the game, it will also resize to fullscreen when the game changes size or orientation.
 
+Flame provides two kinds of `ParallaxRenderer`: `ParallaxImage` and `ParallaxAnimation`, both work
+the same way (`ParallaxImage` is a static image renderer and `ParallaxAnimation` is, as it's name implies, an animation and frame based renderer).
```

Comment:
nit: instead of saying that both work the same way and explaining the difference in parenthesis I would just explain the difference, I don't think "both work the same way" is as relevant as saying the actual difference between them (everything else is assumed to be the same)